protobufs/livekit_rtc_v2.proto

```diff
@@ -68,6 +72,29 @@ message ConnectionSettings {
   bool disable_ice_lite = 4;;
 }
 
+message TrackSettings {
+  string cid = 1;
+  string name = 2;
+  TrackSource source = 3;
+  bool muted = 4;
+  Encryption.Type encryption = 5;
+  string stream = 6;
+}
+
+message AudioTrack {
+  TrackSettings track_settings = 1;
+  repeated AudioTrackFeature audio_features = 5;
+  bool disable_red = 6;
+}
+
+message VideoTrack {
+  TrackSettings track_settings = 1;
+  VideoLayer.Mode layer_mode = 2;
+  repeated SimulcastCodec simulcast_codecs = 3;
+  BackupCodecPolicy backup_codec_policy = 4;
+  string sid = 5;  // used when publishing a new codec to an existing track
+}
```

Comment:
Splitting audio/video into separate messages in an attempt to make things cleaner, i. e. to have only relevant fields.